test/scripts/run_e2e_tests.sh

```
@@ -141,72 +144,69 @@ if [[ "${E2E_ENVIRONMENT}" == "quadlet" ]]; then
     echo "Node ${GINKGO_NODE} will run the following tests:"
     cat "${NODE_TESTS}"
 
-    # Combine all tests for this node into a single focus pattern
-    # Use regex OR (|) to match any of the tests
-    if [[ -s "${NODE_TESTS}" ]]; then
-        # Read all tests, escape regex metacharacters, and join them with | for regex OR
-        FOCUS_PATTERN=$(sed 's/[[\.*^$()+?{|\\]/\\&/g' "${NODE_TESTS}" | paste -sd '|')
-        echo "Focus pattern for node ${GINKGO_NODE}: ${FOCUS_PATTERN}"
-        GINKGO_FOCUS="${FOCUS_PATTERN}"
-    fi
+    FOCUS_PATTERN=$(sed 's/[[\.*^$()+?{|\\]/\\&/g' "${NODE_TESTS}" | paste -sd '|')
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)